src/OrchardCore.Modules/OrchardCore.Workflows/Email/Startup.cs

```diff
@@ -0,0 +1,27 @@
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using OrchardCore.Modules;
+using OrchardCore.Workflows.Email.Activities;
+using OrchardCore.Workflows.Email.Drivers;
+using OrchardCore.Workflows.Helpers;
+
+namespace OrchardCore.Workflows.Email
+{
+    [Feature("OrchardCore.Workflows.Email")]
+    [RequireFeatures("OrchardCore.Liquid")]
```

Comment:
`[RequireFeatures("OrchardCore.Workflows")]` I assume. Liquid would not make sense.

Reply:
You're right.

Reply:
Fixed.